spec/openapi.yaml

```diff
@@ -732,6 +736,14 @@ components:
               description: The size of the file in bytes.
               example: 873151
               nullable: true
+            refs:
+              type: object
+              properties:
+                resource:
+                  type: string
+                thumbnailBig:
```

Comment:
Why do you need to get path to the thumbnail in api/job? The path is already in https://github.com/prusa3d/Prusa-Connect-Local/blob/master/spec/openapi.yaml#L843

Reply:
It's because the preview information is taken from /api/job. 
This lets it be easy to show the preview for any client and it's unnecessary to search in the cache.